src/Sylius/Behat/Service/SessionService.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service;
+
+use Behat\Mink\Mink;
+
+final class SessionService implements SessionServiceInterface
+{
+    private Mink $mink;
+
+    private SharedStorageInterface $sharedStorage;
+
+    private SecurityServiceInterface $securityService;
+
+    private const SESSION_CHROME_HEADLESS_SECOND = 'chrome_headless_second_session';
```

Comment:
Does it need to be the same as the session configured in `behat.yml.dist`? If yes, we should probably parameterize it to not hardcode any configuration values... We can parametrize it anyway 😄

Reply:
this version of driver was created only to cover single scenario, as it is for behats purpouse only there is no need to parameter it for now.